.github/workflows/build_test_deploy.yml

```diff
@@ -735,6 +735,7 @@ jobs:
           path: packages/next-swc/crates/wasm/pkg-nodejs
 
       - run: ls packages/next-swc/crates/wasm
+        if: ${{needs.build.outputs.docsChange != 'docs only change'}}
```

Comment:
Is there anyway we could invert these conditions so they don't need to be repeated for every step of every yaml?

Reply:
Currently not that we've found as we need the jobs to show as successful for the branch protection rules/kodiak to work correctly.